src/main/java/stringCalculator/Operator.java

```diff
@@ -0,0 +1,28 @@
+package stringCalculator;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum Operator {
+    PLUS("+", (x, y) -> x + y),
+    MINUS("-", (x, y) -> x - y),
+    MULTIPLE("*", (x, y) -> x * y),
+    DIVIDE("/", (x, y) -> x / y);
+
+    private final Op op;
+    private final String operator;
```

Comment:
인스턴스 변수 순서는 되도록 통일하는게 좋습니다:)
```suggestion
    private final String operator;
    private final Op op;
```

Reply:
통일하여 사용하겠습니다!